internal/formatter/yaml_formatter.go

```diff
@@ -126,3 +134,24 @@ func mergeNodes(node *yaml.Node) {
 
 	node.Content = node.Content[:k]
 }
+
+func quoteKeysWithDot(node *yaml.Node) {
```

Comment:
Started applying this to test files, but this could also have unexpected results.

This is not used now, if we finally decide not to use it I will remove it.

Reply:
Not very confident to apply this transformation. Maybe just in transform.yml files ?

Reply:
Yeah, after some tests I think this one can be a bit misleading, I am more leaned to remove it, and apply only the automation for nested objects in manifests.

Reply:
Removed